posthog/session_recordings/session_recording_api.py

```diff
@@ -360,7 +370,12 @@ def persist(self, request: request.Request, *args: Any, **kwargs: Any) -> Respon
         return Response({"success": True})
 
     @extend_schema(exclude=True)
-    @action(methods=["GET"], detail=True, renderer_classes=[SurrogatePairSafeJSONRenderer])
+    @action(
+        methods=["GET"],
+        detail=True,
+        renderer_classes=[SurrogatePairSafeJSONRenderer],
+        throttle_classes=[SnapshotsBurstRateThrottle, SnapshotsSustainedRateThrottle],
+    )
     def snapshots(self, request: request.Request, **kwargs):
```

Comment:
I'd love to have a grafana graph of calls to this API by personal API token
I don't know if that's something we already have somewhere or would need to add
But want to understand how much this is being used before it becomes the new events API being used for back door export

Reply:
I could only find when we track if the endpoint is throttled: https://github.com/PostHog/posthog/blob/1eea80b8522c5f9c27867329390052b4c734d806/posthog/rate_limit.py#L20C1-L24

Added specific graph for this endpoint when accessed via a personal API key in [`eb52e49` (#24502)](https://github.com/PostHog/posthog/pull/24502/commits/eb52e49201cc979ebb8b4347c0d5359e5a77ee74)